Src/Base/AMReX_FabArray.H

```diff
@@ -472,6 +472,18 @@ public:
                CpOp                 op = FabArrayBase::COPY)
         { ParallelCopy(fa,period,op); }
 
+    void ParallelAdd_nowait(const FabArray<FAB>& fa,
+                            const Periodicity& period = Periodicity::NonPeriodic())
+       { ParallelCopy_nowait(fa,period,FabArray::ADD); }
+    void ParallelCopy_nowait (const FabArray<FAB>& fa,
+                              const Periodicity&   period = Periodicity::NonPeriodic(),
+                              CpOp                 op = FabArrayBase::COPY)
+       { ParallelCopy_nowait(fa,0,0,nComp(),0,0,period,op); }
+    void copy_nowait (const FabArray<FAB>& fa,
```

Comment:
We don't need provide `copy_nowait`.  We only kept `FabArray::copy` for backward compatibility.